TYPO3.Neos/Classes/TYPO3/Neos/Aspects/NodeTypeConfigurationEnrichmentAspect.php

```diff
@@ -382,7 +382,11 @@ protected function translateAndConvertHelpMessage(array &$configuration, $idPref
     protected function addTargetAttribute($htmlString)
     {
         $document = new \DOMDocument();
-        $document->loadHTML($htmlString);
+
+        // Force correct unicode handling
+        $document->loadHTML('<?xml encoding="UTF-8">' . $htmlString);
+        $document->encoding = 'UTF-8';
```

Comment:
not sure if it's an issue here but in a related comment the author removes the encoding tag afterwards: http://php.net/manual/en/domdocument.loadhtml.php#95251